pipelines/wdl/arrays/imputation_beagle/input_qc/test_inputs/Plumbing/NA12878_qc_fails_bad_coded_indels.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ArrayImputationQC.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation_beagle/input_qc/scientific/vcfs/test_qc_bad_coded_indels.vcf.gz",
+  "ArrayImputationQC.ref_dict": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+  "ArrayImputationQC.reference_panel_path_prefix": "gs://broad-gotc-test-storage/imputation_beagle/scientific/1000G_HGDP_no_singletons_reference_panel/hgdp.tgp.gwaspy.AN_added.bcf.ac2",
+  "ArrayImputationQC.contigs": ["chr1","chr2","chr3","chr4","chr5","chr6","chr7","chr8","chr9","chr10","chr11","chr12","chr13","chr14","chr15","chr16","chr17","chr18","chr19","chr20","chr21","chr22"],
```

Comment:
I'm not sure why the error message you're seeing includes a message about no variant data in supported contigs "chrX" and "chrY". it should be looking for these contigs here. ah I think your screenshot came from a different run - I see this in the warp test: 
<img width="627" height="138" alt="image" src="https://github.com/user-attachments/assets/58c17937-49fd-40dd-9a55-2a5d2788687c" />

Reply:
looks great, thank you!